lib/connection.js

```diff
@@ -1,12 +1,126 @@
 'use strict';
 
 const common = require('metarhia-common');
-const EventEmitter = require('events');
+const { EventEmitter } = require('events');
 
-const Connection = function() {
+const parser = require('./parser.js');
+
+const Connection = function(transport) {
   EventEmitter.call(this);
+
+  this._transport = transport;
+  this._parcels = [];
+  this._chunks = [];
+
+  // Mechanism for collecting full data,
+  // cut by lower-level protocols
+  this._bytesToRead = 36;
```

Comment:
Magic number, need named constants

Reply:
Maybe we should export constants like `HANDSHAKE_LENGTH`, `PARCEL_LENGTH` and `CHUNK_LENGTH` from parser?